### DISCUSSION TITLE
Exporting element values into csv files

### DISCUSSION THREAD

**Discussion by sinaomrani96** [idaholab/moose#27584] [Q&A General] [answered]
### Check these boxes if you have followed the posting rules.

- [X] Q&A General is the most appropriate section for my question
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (screenshots as a last resort, triple back quotes around pasted text)

### Question

Dear all,

I was wondering if there is any postprocessors or vector postprocessors that can exports element values for all the elements. Now I am doing it by Paraview, which it seems not to be the best way to do it specially for big meshes. Maybe something similar to LineValueSampler which reads a csv file of lines coordinates or ElementalVariableValue if it can be extended for the entire elements. I would appreciate it if you could help me. Thanks.

**Comment by GiudGiud** [comment, marked as answer]
Hello

You mean this?
https://mooseframework.inl.gov/source/vectorpostprocessors/ElementValueSampler.html

Guillaume

**Reply by sinaomrani96** [reply]
Hi,

Thank you for the prompt response. Yes, that's exactly what I need. Thank you.
